### PR TITLE
Adding import-bulk command to the documentation

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -659,6 +659,29 @@ mattermost import
   Child Command
```
Comment (amyblais) [inline source/administration/command-line-tools.rst]: Please include this also up here in the table of contents section.

Comment (jespino) [reply source/administration/command-line-tools.rst]: Done, thanks! :)

**Comment by wiersgallak** [comment]
@jespino I see you have a dev review label on this and George was on it and then removed. Does this still need dev review?

**Comment by jespino** [comment]
@wiersgallak I think so, I think was removed by accident.

**Pull request merged**